roles/mysql-server/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+  - name: install mysql-server
+    yum:
+      name: mysql-server
+      state: present
+    when: ansible_distribution == 'CentOS'
+
+  - name: install mysql-server
+    apt:
+      name: mysql-server
+      update_cache: yes
+    when: ansible_distribution == 'Debian'
+
+  - name: config to allow local connections only
+    lineinfile:
+      dest: /etc/my.cnf
+      line: 'bind-address = 127.0.0.1'
+      insertafter: '^\[mysqld\]'
+      state: present
+    when: ansible_distribution == 'CentOS'
+
+  - name: config to allow local connections only
+    lineinfile:
+      dest: /etc/mysql/my.cnf
+      line: 'bind-address = 127.0.0.1'
+      insertafter: '^\[mysqld\]'
+      state: present
+    when: ansible_distribution == 'Debian'
+
+  - name: start mysql service
+    service: name=mysqld state=started
```

Comment:
We should also see if we can run the mysql secure installation command: https://dev.mysql.com/doc/refman/5.7/en/mysql-secure-installation.html